### PR TITLE
Homepage/Routing/Request - Amiibo integration (recent releases)

### PR DESCRIPTION
**Pull request by AriZeto**
Basic (work in progress) homepage created. Routing implemented. Made a request to the Amiibo API and integrated data into homepage, developed function to determine most recent releases. Render said content on homepage, some minor styling applied.

### CODE REVIEW

**Comment by tkwonn** [comment]
Looks good. Could you store the api url in env file?

File: src/pages/Home.tsx [hunk 1]
```diff
@@ -1,3 +1,67 @@
...
+                    <Card>
+                        <CardInfo>
+                            <p>{amiibo.name}</p>
+                        </CardInfo>
+                        <InnerCard>
+                            <CardImg src={amiibo.image} />
+                        </InnerCard>
+                    </Card>
+                ))}
+            </Wrapper>
```
Comment (tkwonn) [inline src/pages/Home.tsx:64]: And would you separate these UI into individual components? I want to use these when I'm working on /amiibos page :pray:

Comment (AriZeto) [reply src/pages/Home.tsx:64]: Yes, of course. I will do that now (and integrate the API into the .env file)

**Comment by AriZeto** [comment]
See new commit.

**Comment by tkwonn** [comment]
Screenshot 2024-04-17 at 15 53 51

It says ['data'] data is undefined when I run 'npm run dev'

**Review by tkwonn — APPROVED** [review]

**Pull request merged**